test/commons/dom/visually-contains.js

```diff
@@ -47,4 +47,12 @@ describe('dom.visuallyContains', function () {
 		assert.isTrue(axe.commons.dom.visuallyContains(target, target.parentNode));
 	});
 
+	it('should return true when element is inline', function () {
```

Comment:
Couldn't you use position:absolute or float to move inline elements outside their parent? Through clipping an child element can also be outside it's parent. There are probably some other ways to do it too. So I'm not sure the assumption you're making here is right.